### DISCUSSION TITLE
Add onChangeEnd support to ColorPicker and ColorInput

### DISCUSSION THREAD

**Discussion by fadyscube** [mantinedev/mantine#2391] [Feature requests]
### What package has an issue

@mantine/core

### Describe the bug

I am using a controlled ColorInput, and I have noticed that if onChange is called each time the point is moved it will be too slow with a large component, and it will be great to call onChange each time the cursor is clicking outside the component and I've found the fixOnBlur property that is supposing to call onChange in the `blur` event and I've tested this and it is continuing calling onChange every time. Here is a code example.
```jsx
import { useState } from 'react';
import { ColorInput } from '@mantine/core';

function App () {
    const [value, setValue] = useState();

    return <div className="App">
        <ColorInput fixOnBlur={false} value={value} onChange={setValue} />
    </div>
}

export default App;
```

### What version of @mantine/hooks page do you have in package.json?

^5.2.0

### Do you know how to fix the issue

No

### Are you willing to participate in fixing this issue and create a pull request with the fix

No

**Comment by rtivital** [comment]
Hi @fadyscube, `fixOnBlur` works as expected, this prop is used to control whether invalid value should be reset to last known valid value on blur. I guess you want `onChangeEnd` event similar to Slider component. I've added this feature to the backlog, it will be added in one of the future releases.

**Reply by fadyscube** [reply]
Okay, sorry, I didn't understand the documentation. Thank you.
